packages/grid/x-data-grid/src/locales/esES.ts

```diff
@@ -58,46 +58,46 @@ const esESGrid: Partial<GridLocaleText> = {
   filterPanelInputPlaceholder: 'Valor de filtro',
 
   // Filter operators text
-  filterOperatorContains: 'contiene',
-  filterOperatorEquals: 'es igual',
-  filterOperatorStartsWith: 'comienza con',
-  filterOperatorEndsWith: 'termina con',
-  filterOperatorIs: 'es',
-  filterOperatorNot: 'no es',
-  filterOperatorAfter: 'es posterior',
-  filterOperatorOnOrAfter: 'es en o posterior',
-  filterOperatorBefore: 'es anterior',
-  filterOperatorOnOrBefore: 'es en o anterior',
-  filterOperatorIsEmpty: 'está vacío',
-  filterOperatorIsNotEmpty: 'no esta vacío',
-  filterOperatorIsAnyOf: 'es cualquiera de',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  filterOperatorContains: 'Contiene',
+  filterOperatorEquals: 'Es igual',
+  filterOperatorStartsWith: 'Comienza con',
+  filterOperatorEndsWith: 'Termina con',
+  filterOperatorIs: 'Es',
+  filterOperatorNot: 'No es',
+  filterOperatorAfter: 'Es posterior',
+  filterOperatorOnOrAfter: 'Es en o posterior',
+  filterOperatorBefore: 'Es anterior',
+  filterOperatorOnOrBefore: 'Es en o anterior',
+  filterOperatorIsEmpty: 'Está vacío',
+  filterOperatorIsNotEmpty: 'No esta vacío',
+  filterOperatorIsAnyOf: 'Es cualquiera de',
+  'filterOperator=': 'es igual a',
+  'filterOperator!=': 'es diferente a',
+  'filterOperator>': 'es mayor que',
+  'filterOperator>=': 'es mayor o igual que',
+  'filterOperator<': 'es menor que',
+  'filterOperator<=': 'es menor o igual que',
```

Comment:
From what I can see, these need to stay in mathematical notation, there is nothing to translate here. Not sure but that's what I see elsewhere.